### DISCUSSION TITLE
Thoughts on Food

### DISCUSSION THREAD

**Discussion by RadCatish** [FakeFishGames/Barotrauma#4835] [Ideas and suggestions]
The Trello lists "Food and Hunger" as a long-term hope for the game, and it has a lot of votes. https://trello.com/b/hBXI8ltN/barotrauma-roadmap-known-issues

Here are some of my thoughts on this, anyone else got ideas?

- New "Chef" or "Cook" class and skillset
- Low cooking skill means you can't identify if plants are ingredients or poisonous, and don't know recipes
- Classes of food, eg "junk" "healthy" "snack" 
- Vitamin/mineral deficiencies leading to illnesses like scurvy
- Turn ethanol into actual drinks (rum is a must on any sea-faring vessel, surely?)
- Expiry of perishable food (and other organic items such as alien blood and organs, pre-stabilozine), perhaps indicated in the same way fuel rods/oxygen tanks show depletion
- Use the vending machines in outposts! And have restaurants?
- Hydroponic plant growing rooms, with pumps for auto-circulation so you don't have to keep watering them
- Option to buy plenty of freeze-dried, astronaut-style food packs for fairly cheap for those who want to opt-out of the new mechanic
- New plants to gather outside of the sub - great motivation for more EVAs and exploring. Perhaps they get better the deeper/further you go?
- High hunger leads to fainting, loss of vision
- Very low hunger gives boosts? Faster walking speed, faster repair times?

**Comment by Kulik242** [comment]
I respect your opinion, but I disagree with most of those ideas.

I hate to have some arbitrary hunger bar that you constantly need to replenish during a mission, let alone build a class around it and fill it with more arbitrary mechanic.

I see how pissed I could get if I'm doing a fun ruin exploration or wreck salvaging mission and all off sudden my vision gets blurry cause I forgot to bring a snickers bar. Or the ship is sinking, reactor is burning and now we are all going to die because the engineer and mechanic haven't had their orange juice for the past 10 minutes and now due scurvy they can't run and take forever to fix things.

IMO, food should be a consideration between missions. It should certainly be accounted for giving a sense of planing the expedition into the heart of Europa. It could be bought, farmed or foraged giving players the sense that the crew has to sustain itself during the campaign.

I don't mind food having some effects ranging from small boosts to vitality to poisoning (poisoning food as a traitor sounds fun).

The "opt-out of the mechanic by buying astronaut food" solution doesn't really work, cause once you have these mechanics in game, you need to balance the game around them, so the devs either commit and make it a part of the game mechanic or make it a roleplay gimmick.

**Comment by NilanthAnimosus** [comment]
I will say first I do not think food should be added. however I will share my constructive criticism, thoughts, and how it can be implemented potentially well for others to discuss:

This has been touched on many times and not added every time, more so as a hunger system doesn't tend to inherently make a short round-based game any more fun, It becomes a question of how often to eat this arbitrary food and what it does and doesn't do if you eat well or not - but we do have MP campaign and SP campaign working very well.

They removed charging your headsets battery since most found it an additional chore that didn't massively add to the gameplay, many would forget about it, and be punished later for something they didn't notice. Adding a mechanic like hunger to give the same thing they removed is probably unwanted by devs and players alike for a majority.

However, I love my survival games, and food/water and ensuring I can get food in them is a thing I do like, but it needs to be unobtrusive, even an "Opt-Out" solution that costs little and requires you to fetch a thing and eat it X times a round is still just an additional chore that isn't directly fun or adding to the game beyond grabbing an item to use periodically and a money sink.

My take on food not being annoying in a sense making way if it was actually wanted to penalize your games this way with extra difficulty  would be very different:

Food items are not consumed in game, they are items with condition that you store, such as food boxes and supplies and such, there is no water supplies mechanics as your in an ocean surrounded by salt water anyone could process so just toss that idea out because its just adding chores.

There must be an easy way to see and understand every round food stocks by type and hunger effects. I suggest adding to the Crew/Mission/Character tabs in the crew list or an additional tab for supplies detailing submarine inventory (Just at round start by quantity w/ filter, nothing processor intensive or such) so this is not too easy to miss or annoying to check.

--There is no job for 'cooking', honestly I would not even give it a skill. the AI do not craft for me and I don't want to be forced to play a glorified chef, sounds like the worst role honestly.

There is no malice for 1-2 rounds if you do not manage to obtain food supplies once you run out. starvation takes time, no one round sudden debuffs. At most the first 1-2 rounds NPC dialogue should complain of the lack of food and additionally converse about it in their conversations, adding to the flair of the game with their respective personality traits.

Beyond 2-3 rounds of no food supplies, maximum vitality decreases by around 3 to 10 for every round without food (Preferably outside of outposts only to not double-consume food supply, as locations without outposts skip ). you don't strictly "Lose health" as much as you lose your strength gradually taking less to die with indicators for the afflictions just like damage/bleeding etc.

As a note on the above, Captains start with 80 vitality points, other jobs at 90 and security/doctors get 100 for reference on maximum vitality aka health to understand the effect of this, it also means an uninjured captain would take between around 8 to 31 rounds to starve to death impacted heavily by untreated injuries.

As an alternative to the above, a reverse-resistance modifier given by hunger that makes you more susceptible to everything but oxygen loss may be a better alternative that strength is based on your hunger, and after X rounds worth of strength, causes vitality to drop (But not maximum vitality) - eventually killing your crew that way without directly killing them initially but causing the same innate weakness.

Beyond 6 rounds perhaps integrate minor scaling speed penalties as well. This can all be done with affliction strength presumably scaled on round start by setting character afflictions (Or just adding/removing them progressively at round starts), I would not make this greater than 25% reduction for game play reasons as nobody wants to move super sluggish.

If having food qualities (By different kinds of food with different expenses) - Perhaps eat best-first or choose in the Crew/submarine supplies UI tab (if one is added to the crew/mission/character in game menu) which food to use next round, and grant different bonuses on food type/quality.

I would keep these none-magical and fairly simple, such as a very tiny regeneration of health to a cap, a slightly slower death from oxygen affliction when downed, or just a small boon to vitality (Such as +2 / round going up to +10 or + 20 depending on food type, decreasing fast when changing to a worse food) General resistance afflictions from food could be another way to get the same rough effect w/ indication to the player.

I feel food should be kept out of the round beyond storing it, avoiding its destruction or loss, and other such. Perhaps some foods cannot survive water well (Soggy saltwater bread and crackers anyone?), and others are fairly bulky to quantity, and some could be cheap and high-stocks.

We don't have wages taking money out of our pockets, but food supplies could act as them, a scaling cost to crew size and additional consideration when shopping and reason for additional kinds of hydroponics and plant-life / edible loot when processed from monsters. It could and should be used to add depth and more reason to want good cash flow, and more preparation when going deep into Europa away from the colonies.

Either way, I feel if food is implemented, it should go through an extensive design phase and come as one complete feature in a large update, and not trickled in as updates with gaping holes or irritations to current game play. it can work, but should be thought out well and discussed thoroughly.

**Reply by jamesjlee1211-ui** [reply]
I really like how this discussion explores making food systems more meaningful things like cooking skill, perishable items, and bonuses/penalties based on diet are all interesting. The suggestion about food classes (junk, healthy, snack) as well as vitamin/mineral deficiencies (e.g. scurvy) adds depth.

One concern though: you want mechanics that add strategy without becoming a burdensome chore. For example, rather than forcing constant eating, having a “meal planning / menu” UI that lets you stock and prepare in advance could reduce micromanagement.

On the topic of “menu variety as incentive,” I often look to real-world menu systems for inspiration comparing options, balancing cost, nutrition, flavor. When I’m planning out real meals, I use [Subway Menu Prices](https://subwaymenuz.com/)
to browse subway menu with prices ahead of time. It’s a small feature, but seeing real menu variety helps me imagine how to design interesting food systems in a game context.
